### ISSUE TITLE
[HW] Update planning doc "software bill of materials"

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#6]
## User story
1. As a project team
2. I need to update the planning doc "software bill of materials" tab
3. So that a record of all technology dependencies are kept track of and updated as they change

## Acceptance criteria
* All selected technologies/tools need to satisfy the course licensing requirements (MIT license).

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
